src/5e-SRD-StartingEquipment.json

```diff
@@ -1337,5 +1337,45 @@
       }
     ],
     "url": "/api/starting-equipment/wizard"
+  },
```

Comment:
I just remembered that we have an [open issue](https://github.com/5e-bits/5e-database/issues/251) from @fergcb to combine StartingEquipment into Classes. So this may also be better to just be folded into Backgrounds since this is also a 1:1 relationship.

Reply:
Which finally got my butt in gear to put together [this PR](https://github.com/5e-bits/5e-database/pull/343).